src/Features/CSharp/Portable/Completion/KeywordRecommenders/ProtectedKeywordRecommender.cs

```diff
@@ -62,12 +62,11 @@ private static bool IsValidContextForType(CSharpSyntaxContext context, Cancellat
 
         private static bool CheckPreviousAccessibilityModifiers(CSharpSyntaxContext context)
         {
-            // We can show up after 'internal'.
+            // We can show up after 'internal' and 'private'.
```

Comment:
Good catch, thanks.
Pushed a commit to address that. Per discussion with Neal, we should do the same as with "internal protected" (ie. allow it in either order).